basin3d/core/synthesis.py

```diff
@@ -56,7 +56,66 @@ def extract_id(identifer):
             if x.startswith("{}-".format(id_prefix))]
 
 
-class DataSourceModelIterator(Iterator):
+class MonitorMixin(object):
+    """
+    Adds monitor log functionality to a class for loggind synthesis messages
```

Comment:
typo: loggind --> logging

Reply:
Good catch. btw. if you make  suggested corrections in the github ui for PR reviews then I can easily add the change to the PR from the UI.